### PR TITLE
Add content type header for patch

### PR DESCRIPTION
**Pull request by eedwards0**
Add header required for patch

### CODE REVIEW

File: test/services/psc-verification-link/service.spec.ts [hunk 1]
```diff
@@ -137,5 +138,51 @@ describe("PSC Verification Link", () => {
...
+        it("should return a status 200 OK and patched PSC individual verification filing when the 'content-type' is set to 'application/merge-patch+json'", async () => {
+            const patchJsonHeader = {
+                name: "Content-Type",
+                value: "application/json"
+            }
+
+            const scope = nock(/.*/)
+                .patch("/")
+                .matchHeader(patchJsonHeader.name, patchJsonHeader.value)
+                .reply(415);
```
Comment (PhilipGlover) [inline test/services/psc-verification-link/service.spec.ts:151]: what is this `scope` for ?
I've commented it out and the test still runs and passes

File: test/services/psc-verification-link/service.spec.ts [hunk 2]
```diff
@@ -137,5 +138,51 @@ describe("PSC Verification Link", () => {
...
+        it("should return a status 415 unsupported media type when the 'content-type' is not 'application/merge-patch+json'", async () => {
+            const applicationJsonHeader = {
+                name: "Content-Type",
+                value: "application/json"
+            }
+
+            const scope = nock(/.*/)
+                .patch("/")
+                .matchHeader(applicationJsonHeader.name, applicationJsonHeader.value)
+                .reply(415);
```
Comment (PhilipGlover) [inline test/services/psc-verification-link/service.spec.ts:174]: same as above

File: src/services/psc-verification-link/service.ts [hunk 3]
```diff
@@ -10,7 +10,8 @@ export default class PscVerificationService {
 
     public async postPscVerification (transactionId: string, pscVerification: PscVerification): Promise<Resource<PscVerificationResource> | ApiErrorResponse> {
         const resourceUri = `/transactions/${transactionId}/persons-with-significant-control-verification`;
-        const response = await this.client.httpPost(resourceUri, pscVerification);
+        const additionalHeaders = { "Content-Type": "application/merge-patch+json" };
+        const response = await this.client.httpPost(resourceUri, pscVerification, additionalHeaders);
```
Comment (PhilipGlover) [inline src/services/psc-verification-link/service.ts:14]: wait this header has been placed on the `postPscVerification`.  It needs to go on `patchPscVerification` not this one.

File: test/services/psc-verification-link/service.spec.ts (hunk 1 above)
Comment (eedwards0) [reply test/services/psc-verification-link/service.spec.ts:151]: My intent was to explicitly set the content-type header in the request.

File: test/services/psc-verification-link/service.spec.ts [hunk 4]
```diff
@@ -6,6 +6,7 @@ import PscVerificationService from "../../../src/services/psc-verification-link/
 import { PscVerificationResource } from "../../../src/services/psc-verification-link/types";
 import Resource, { ApiErrorResponse } from "../../../src/services/resource";
 import { COMPANY_NUMBER, FILING_ID, PSC_VERIFICATION_CREATED, PSC_VERIFICATION_ID, PSC_VERIFICATION_IND, PSC_VERIFICATION_RLE, TRANSACTION_ID, mockPscVerificationCreatedResource, mockPscVerificationCreatedResponse, mockPscVerificationIndResource, mockPscVerificationIndResponse, mockPscVerificationPatchIndResource, mockPscVerificationPatchIndResponse, mockPscVerificationPatchRleResponse, mockPscVerificationPatchRleRoResource, requestClient } from "./service.mock";
+import nock from "nock";
```
Comment (PhilipGlover) [inline test/services/psc-verification-link/service.spec.ts:9]: not used

Comment (eedwards0) [reply test/services/psc-verification-link/service.spec.ts:9]: 👍

**Review by PhilipGlover — APPROVED** [review]
- additional Header to Patch as required
- have run this change elsewhere and I successfully used it to update the resource in the collection
- tests pass
```
  PSC Verification Link
    POST endpoint
      ✔ should return status 201 Created and filing resource representation on authorised access
      ✔ should return status 401 Unauthorised on unauthorised access
      ✔ should return staus 400 Bad Request for bad data
    GET endpoint
      ✔ should return status 200 OK and filing resource representation on authorised access
      ✔ should return status 401 Unauthorised on unauthorised access
      ✔ should return status 404 Not Found if resource id not found
    PATCH endpoint
      ✔ should return a status 200 OK and patched PSC individual verification filing
      ✔ should return a status 200 OK and patched PSC RLE verification filing
      ✔ should return a status 401 Unauthorised on unauthorised access
      ✔ should return a status 404 Not Found if resource id not found
      ✔ should return a status 415 unsupported media type when the 'content-type' is not 'application/merge-patch+json'
```

**Pull request merged**